docs/modules/idrac_secure_boot.rst

```diff
@@ -135,7 +229,10 @@ Notes
 
 .. note::
    - This module will always report changes found to be applied when run in \ :literal:`check mode`\ .
-   - This module does not support idempotency when \ :emphasis:`import\_certificates`\  is provided.
+   - This module does not support idempotency when \ :emphasis:`reset\_type`\  or \ :emphasis:`export\_certificates`\  or \ :emphasis:`import\_certificates`\  is provided.
+   - The order of operations is as follows configure Secure Boot settings(boot\_mode, secure\_boot, secure\_boot\_mode, secure\_boot\_policy, force\_int\_10), export, certificate reset, import, followed by idrac reset.
```

Comment:
To configure the secure boot settings, the idrac_secure_boot module performs the following order of operations  set attributes, export certificate, reset keys, import certificate, and restart iDRAC.

Reply:
Updated